### ISSUE TITLE
Our multiple `nav` elements should have aria-labels

### ISSUE THREAD

**Issue by macdonst** [enhance-dev/enhance.dev#54]
They should have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable
